### ISSUE TITLE
Business icons

### ISSUE THREAD

**Issue by mika76** [FortAwesome/Font-Awesome#818]
The icons right now are absolutely brilliant, but they mostly supply the generic web app design. There is still a lack of icons and glyphs for business and industry. 

For example I'm in the casino industry and there are always a lack of icons for slot machines, different table games, currencies, business actions like counting money and bill validators, signs, jackpots, etc...

Now I understand the casino industry is a bit of a fringe case (and I use the example only because it's the one I see regularly), but this goes for quite a few industries and businesses,

So basically my request is for more "items" required for business apps. Different machines, vehicles, people types, book types, etc...
